### ISSUE TITLE
Webside not loaded (isotope product-list) 

### ISSUE THREAD

**Issue by Ernestopheles** [cytopia/devilbox#949]
### Have you already looked into this bug?

- [X] I have checked the [Troubleshooting Guide](https://devilbox.readthedocs.io/en/latest/support/troubleshooting.html)
- [X] I have checked [Release notes](https://github.com/cytopia/devilbox/releases) for potential migration steps
- [X] I have checked [existing issues](https://github.com/cytopia/devilbox/issues) for similar bugs
- [X] I have googled this bug already with no luck
- [ ] I have not done any of the above

### (Optional) Error message

HTTP CODE 503 

### What went wrong?

webpage unreachable

### Expected behaviour

webpage shown, webshop with product list

### How can we reproduce the bug?

see #948; also have isotope 2.8.8. installed and configured

### Host Operating System

Linux

### Host Platform (amd64, arm64, other)

amd64

### (Linux only) Is SELinux enabled?

No, SELinux is disabled

### Docker version

#948

### Docker Compose version

#948

### Devilbox version

#948

### Have you removed stopped containers before starting?

Yes

### Have you pulled latest Docker images before starting?

Yes

### Devilbox start command

#948

### Config: .env file

```shell
#948
```

### Config: ./check-config.sh

```shell
#948
```

### Log: docker-compose logs

```shell
#948
```

### (Optional) Additional information

All webpages are shown, all but the one with the product list

**Comment by Ernestopheles** [comment]
I tried several more configurations, f.i. those from jonatanaxe https://github.com/cytopia/devilbox/issues/946#issuecomment-1372241474

Also tried my former xdebug.ini settings

Also checked each suggested php-option from the contao manual

Nothing changed anything at all.


Interstingly, when I step-debug through, the page in question will be shown!

**Comment by Ernestopheles** [comment]
Also I found - in difference to the relaes notes - that no listening daemons panel is shown.

**Comment by cytopia** [comment]
> I tried several more configurations, f.i. those from jonatanaxe [#946 (comment)](https://github.com/cytopia/devilbox/issues/946#issuecomment-1372241474)
> 
> Also tried my former xdebug.ini settings
> 
> Also checked each suggested php-option from the contao manual
> 
> Nothing changed anything at all.
> 
> Interestingly, when I step-debug through, the page in question will be shown!

Is this a Xdebug issue?

**Comment by cytopia** [comment]
> Also I found - in difference to the relaes notes - that no listening daemons panel is shown.

This is only shown if you have any custom listeners running on your PHP container. If there are none, then it is not shown.

**Comment by Ernestopheles** [comment]
No, i rather think thats not a xdebug issue but a devilbox issue.

As Problem does Not show Up in previous Release, Shows Up in Run Mode and Not in debug Mode and has s th. to do with the xdebug settings in php.ini - this makes me think it IS a Side effect of the new Release .

**Comment by cytopia** [comment]
The xdebug settings are not enable by default. You have to actively copy the configuration file in order to enable it. Is that the case with your setup?

**Comment by Ernestopheles** [comment]
Yes, I added this file to the corresponding cfg folder:

xdebug.ini:

`; ############################################################
; # Devilbox xdebug.ini: individuelle php settings für vscode Xdebug
; ############################################################
;
; PHP.ini configuration
;  
[PHP]
; Defaults
;xdebug.mode                 = debug
xdebug.remote_handler       = dbgp
xdebug.start_with_request   = yes

; How to connect
xdebug.client_port          = 9003
xdebug.client_host          = host.docker.internal
xdebug.discover_client_host = false

; Logging
xdebug.log                  = /shared/httpd/xdebug.log
xdebug.log_level            = 7

; IDE Configuration
xdebug.idekey               = VSCODE

`

**Comment by Ernestopheles** [comment]
The semicolon before the first statement is my workaround to have my webpage running. When I want to use xdebug I have to uncomment this line

**Comment by Ernestopheles** [comment]
So, changed DNS server Address in Router to 1.1.1.1 / 1.0.0.1 and corr. ipv6 addresses.

browsing side https://cloudflare-dns.com/help/ I get positive result and this URL:

https://cloudflare-dns.com/help/#eyJpc0NmIjoiWWVzIiwiaXNEb3QiOiJObyIsImlzRG9oIjoiTm8iLCJyZXNvbHZlcklwLTEuMS4xLjEiOiJZZXMiLCJyZXNvbHZlcklwLTEuMC4wLjEiOiJZZXMiLCJyZXNvbHZlcklwLTI2MDY6NDcwMDo0NzAwOjoxMTExIjoiWWVzIiwicmVzb2x2ZXJJcC0yNjA2OjQ3MDA6NDcwMDo6MTAwMSI6IlllcyIsImRhdGFjZW50ZXJMb2NhdGlvbiI6IkhBTSIsImlzV2FycCI6Ik5vIi

BUT: no connection to contao413.dvl.to by browser access or ping or traceroute.

"ping: contao413.dvl.to: Zu diesem Hostnamen gehört keine Adresse"

Opened a ticket at avm (router company), maybe should open a ticket with my provider (Telekom)?

**Comment by Ernestopheles** [comment]
Tried with the local installation (https://www.isotopeecommerce.org/en/demo.html) of the [isotope demo website](https://demo.contao.way-vision.de/), same error:

The Isotope demo behaves accordingly: all pages with the list module cannot be accessed (503), while the other pages can. It would be great if we could get to the bottom of the problem by trying to answer the question: What could cause a web server to not be able to access pages with the list module? Since the xdebug mode itself runs and the pages can also be debugged, it must be a side effect that only occurs when xdebug is enabled in php.ini but not used.

Also hoping for some ideas by the contao community.

**Comment by Ernestopheles** [comment]
Now I am using docker desktop and hence have access to more information. In the docker desktop panel / Container Devilbox / View Details I get this error each time a website with Isotope Listmodule is requested:

> [Warning] Aborted connection 9 to db: 'contao413' user: 'root' host: '172.16.238.10' (Got an error reading communication packets)

**Comment by Ernestopheles** [comment]
Further testing results in these log entries, too:

apache 2.4 error.log (when calling the website wirh the listmodul):

> [Fri Feb 24 19:17:10.138802 2023] [proxy_fcgi:error] [pid 2874:tid 140702354287416] [client 172.16.238.1:39884] AH01067: Failed to read FastCGI header, referer: http://contao413.dvl.to/impressum.html
[Fri Feb 24 19:17:10.155147 2023] [proxy_fcgi:error] [pid 2874:tid 140702354287416] (104)Connection reset by peer: [client 172.16.238.1:39884] AH01075: Error dispatching request to : , referer: http://contao413.dvl.to/impressum.html

What I always get in xdebug.log (not dependant on specific webside called):

> [3425] Log opened at 2023-02-24 19:17:02.820985
[3425] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[3425] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[3425] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).

**Comment by Ernestopheles** [comment]
From the Contao side there is nothing they can help with. 

As I did not use Isotope before I can not say if the issue is new since Devilbox 3.

Unfortunately I am at the end of my research and try and error with configuations of php and mysql did not help either.
